app/models/rss_log.rb

```diff
@@ -287,7 +287,7 @@ def url
       format("/glossary/show_glossary_term/%d?time=%d",
              glossary_term_id, updated_at.tv_sec)
     elsif article_id
-      format("/article/show_article/%d?time=%d", article_id, updated_at.tv_sec)
+      format("/articles/%d?time=%d", article_id, updated_at.tv_sec)
```

Comment:
Is the `url` method dead code? 
Is it called in `rss.xml.builder`?
If it's dead, then this PR should be closed and ignored.